go.sum

```diff
@@ -26,6 +26,7 @@ github.com/aristanetworks/goarista v0.0.0-20170210015632-ea17b1a17847/go.mod h1:
 github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6/go.mod h1:grANhF5doyWs3UAsr3K4I6qtAmlQcZDesFNEHPZAzj8=
```

Comment:
Was there no corresponding go.mod change? Or is this just a refresh of our current deps?

Reply:
This is just what `make build` did to it.